src/modules/base/header/header.html

```diff
@@ -6,7 +6,7 @@
             <span class="icon-bar"></span>
             <span class="icon-bar"></span>
         </button>
-        <a class="navbar-brand" ui-sref="dashboard" class="nav-brand-name-link">
+        <a class="navbar-brand" ui-sref="landing-page" class="nav-brand-name-link">
```

Comment:
@gnehapk The pointer is automatically added. I tested it just now the pointer is coming.